scripts/funcTests/runFuncTests.sh

```diff
@@ -135,7 +158,7 @@ echo "Core tests finished at `date -u +"%Y-%m-%dT%H:%M:%S"`"
 
 # Func tests
 echo "dotnet msbuild build/build.proj /t:CoreFuncTests /p:VisualStudioVersion=16.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta"
-dotnet msbuild build/build.proj /t:CoreFuncTests /p:VisualStudioVersion=16.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
+$DOTNET msbuild build/build.proj /t:CoreFuncTests /p:VisualStudioVersion=16.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
```

Comment:
The .NET SDK installed into the `cli/` folder is used by `Dotnet.Integration.Tests`. I don't think we want to use it to build the product though.

Reply:
So, we use System installed .NET SDK to build the product and local one to test the product, right?

Reply:
We don't use the local SDK to run `dotnet test`, we still use the system SDK for that. Rather, [Dotnet.Integration.Test's test fixture](https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs) copies the SDK from the `cli/` folder to a `.test\<random>` folder and then replaces all the NuGet assemblies, then uses it itself.  Ideally, `Dotnet.Integration.Test` would download the SDK itself, so when `Dotnet.Integration.Test` is not run, there's no need to download the SDK into the `cli/` folder. But we're not there yet. In case you feel interested in implementing this, please do not do it in this PR.

I think the script to download the .NET SDK into the `cli/` folder before running other commands was from the pre-.NET Core 1.0 days, when CI agents did not have the .NET SDK pre-installed.  Now that they do, I believe these build/run tests scripts should use the system `dotnet` for everything, except for the `cli/dotnet --info` that helps us validate the build is working as expected.

But leaving each command to use whichever dotnet it's already using is a safe bet, as it minimizes unintended consequences. Let's focus our efforts on improvements that fix reported problems.

Reply:
> We don't use the local SDK to run `dotnet test`

That, I understand. Some calls in the script had `$DOTNET` instead. That's why I was confused. Thought, that might be the problem. I can reproduce the issue in my WSL Ubuntu setup. I don't have a Mac. So, I don't know how to test in a Mac environment. But since both are Unix Like environments, fixing in Linux should do the same in Mac, I think!

> ...please do not do it in this PR...

Don't worry! This patch is complicated already!! 😂 But I'm interested in that change. I'll look into it after this.